### ISSUE TITLE
If the current track has no artwork then the favicon is not reset

### ISSUE THREAD

**Issue by dhowden** [dhowden/tchaik#9]

**Comment by solarnz** [comment]
If the favicon is a 404, it won't update the image.

**Comment by dhowden** [comment]
Yes, so I guess there are two issues here: if there is no track selected OR if there is no artwork then the icon should be reset to the default value.

With the empty selection I tried removing the `href` attribute on the favicon `<link>` tag but it didn't clear it... Tried a few more things without success, but I'm sure there will be a way.

**Comment by solarnz** [comment]
The problem is that once the browser gets a valid favicon, then it will continue to use that if the favicon url cannot be found (either 404, or the element doesn't exist).

I think we will need a default image to show in place.

Maybe we could hit two issues on the head at the same time if we have a default album art image for those files that don't have one set?
